src/UI/Implementation/Component/Input/Input.php

```diff
@@ -310,6 +359,24 @@ private function applyOperationsTo($res) {
 	}
 
 	/**
+	 * Get the operations that should be performed on the input.
+	 *
+	 * @return (Transformation|Constraint)[]
```

Comment:
Is this really an array, I think correct is Generator, isn't it?

Reply:
Indeed...